word_segmentation.py

```diff
@@ -196,86 +205,84 @@ def clean_line(line):
     return line
 
 
-def preprocess():
+def preprocess_Thai(demonstrate):
     """
     This function uses the BEST data set to
         1) compute the grapheme cluster dictionary that holds the frequency of different grapheme clusters
         2) demonstrate the performance of icu word breakIterator and compute its accuracy
+    Args:
+        demonstrate: shows if we want to see how the algorithm is working or not
     """
+    words_break_iterator = BreakIterator.createWordInstance(Locale.getUS())
```

Comment:
I think in this situation you probably want `Locale.getRoot()` instead of `Locale.getUS()`

Reply:
Only for this instance, or generally for all `BreakIterator.createWordInstance` and `BreakIterator.createCharacterInstance`?